.github/workflows/quick-check.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow runs on every push and checks whether everything looks OK
+
+name: Quick Check
+
+on:
+  push:
+    tags:
+      - "!v*"
+    branches:
+      - "*"
```

Comment:
Can you change this to run on pull as well? Then we can double-check that this will work prior to merging and for future pull-requests.

Reply:
It runs on PRs as well thanks to the branch: *
You can test and put this yam file in master and this PR will be checked once I push a change

Reply:
Oh right I see now. I always get confused with these triggers. Everything looks good 👍